CodeStarterCamp_Week2/main.swift

```diff
@@ -45,4 +45,25 @@ func checkLottoNumbers(myLottoNumbers: [Int]) -> Void {
 	}
 }
 
-checkLottoNumbers(myLottoNumbers: myLottoNumbers)
+func saveLottoNumbers(totalCount: Int) -> [String: [Int]] {
+	var savedLotto = [String: [Int]]()
+
+	for count in 1...totalCount {
+		savedLotto["\(count)회차"] = Array(createLottoNumber())
+	}
+	return savedLotto
+}
+
+func showLottoNumbers(totalCount: Int, userCount: Int) {
+	let saveLottos = saveLottoNumbers(totalCount: totalCount)
+	
+	let getCountLotto = saveLottos["\(userCount)회차"]
+
+	guard let getCountLotto else { return print("\(userCount)회차는 아직 진행되지 않았습니다.") }
```

Comment:
딕셔너리의 value는 옵셔널인가보군요!
guard문을 이용하여 잘 바인딩 해주셨습니다!
다만, 이렇게 쓰는 것이 더 자연스러울 것 같습니다.

```suggestion
        guard let getCountLotto else {
            print("\(userCount)회차는 아직 진행되지 않았습니다.")
            return
    }
```

Reply:
아 그러네요.. return이 뒤로 가야하네요!